parley/src/tests/utils/renderer.rs

```diff
@@ -19,7 +19,15 @@ use tiny_skia::{Color, FillRule, Paint, PathBuilder, Pixmap, PixmapMut, Rect, Tr
 
 #[derive(Clone, Copy, Debug, PartialEq)]
 pub(crate) struct ColorBrush {
-    pub(crate) color: Color,
+    pub(super) color: Color,
```

Comment:
Why make this `super` (and why as part of this PR?)

Reply:
I think this was a change that I forgot to undo